src/OpenTelemetry/Trace/Export/ActivityProcessor.cs

```diff
@@ -42,5 +42,12 @@ public abstract class ActivityProcessor
         /// <param name="cancellationToken">Cancellation token.</param>
         /// <returns>Returns <see cref="Task"/>.</returns>
         public abstract Task ShutdownAsync(CancellationToken cancellationToken);
+
+        /// <summary>
+        /// Flushes all activity that have not yet been processed.
```

Comment:
nit: Something about the comment seems off grammatically. Should it be `Flushes all activities that have not yet been processed.` or maybe something like `Flushes all activity objects that have been queued for processing.`

Reply:
Fixed.